extensions/github-pull-requests/src/PullRequestList.tsx

```diff
@@ -0,0 +1,35 @@
+import { List } from "@raycast/api";
+import { useState, useEffect } from "react";
+import fetchPullRequests from "./fetchPullRequests";
+import PullRequestItem from "./PullRequestItem";
+import { PullRequests } from "./types";
+
+export default function PullRequestList() {
+  const [state, setState] = useState<PullRequests>({
+    pullRequests: [],
+  });
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    async function fetch() {
+      const pullRequests = await fetchPullRequests();
+
+      setState((oldState) => ({
+        ...oldState,
+        pullRequests: pullRequests,
+      }));
+
+      setIsLoading(false);
+    }
+
+    fetch();
+  }, []);
+
+  return (
+    <List isLoading={isLoading} searchBarPlaceholder="Filter by name...">
+      {state.pullRequests.map((pr) => (
+        <PullRequestItem key={pr.url} {...pr} />
+      ))}
```

Comment:
Would it make sense to put pull requests into different sections?

Reply:
So that you can see where you got mentioned, assigned and bla.

Reply:
You mean to have a pull request per section? At first I thought that might clutter everything vertically, having so many section headers, but I suppose I could show updated date there and maybe author, then move assignees and reviewers in the accessory.

Reply:
I left a comment on the PR to clarify what I've meant.